runtime/domain.c

```diff
@@ -87,4 +87,8 @@ void caml_init_domain ()
   Caml_state->eventlog_startup_pid = 0;
   Caml_state->eventlog_startup_timestamp = 0;
   Caml_state->eventlog_out = NULL;
+
+  #ifdef NAKED_POINTERS_CHECKER
+  Caml_state->checking_pointer_pc = NULL;
+  #endif
```

Comment:
Is this used under Windows?

Reply:
Right, it is not, I will guard it for _WIN32 as well, thank you.

Reply:
Same for the definition in `domain_state.tbl`.